lib/internal/modules/cjs/loader.js

```diff
@@ -716,9 +716,9 @@ if (isWindows) {
       return [from + 'node_modules'];
 
     const paths = [];
-    var p = 0;
-    var last = from.length;
-    for (var i = from.length - 1; i >= 0; --i) {
+    let p = 0;
+    let last = from.length;
+    for (let i = from.length - 1; i >= 0; --i) {
```

Comment:
nit: move variables `p` and `last` in for loop definition?
```suggestion
    for (let i = from.length - 1, p = 0, last = from.length; i >= 0; --i) {
```

ditto on line 762

Reply:
I personally prefer the current style. Is there any eslint rule we could use to detect cases like these / prohibit one way? I do not know what most other collaborators prefer for this case and opening a separate PR with such a rule might clarify that?

Reply:
> I personally prefer the current style

I prefer the suggested style, as the scope of the variable will be limited to the loop block, and not the outer block.